### ISSUE TITLE
Visibility when logged ingame (BB #828)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#828]
This issue was migrated from bitbucket.
**Original Reporter:** smoldar
**Original Date:** 18.03.2015 11:20:16 GMT+0000
**Original Priority:** minor
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828

<hr>

Not loaded all nearby creatures/objects

When player logged ingame, player see some creatures, but not all. Some creatures are invisible.

After player rotate, or move, after this are loaded other creatures.

I tested log ingame where was in circle near me 25 creatures and i saw only 15-20... after rotate of char or move was loaded other.

Visibility.Distance.Continents = 90
Visibility.Distance.Instances = 170
Visibility.Distance.BGArenas = 350
Visibility.Distance.Object = 50
Visibility.Distance.InFlight = 70
Visibility.Distance.Grey.Unit   = 1
Visibility.Distance.Grey.Object = 10

Screen - Only rotate of character after log into game
![WoWScrnShot_031815_122316.jpg](https://bitbucket.org/repo/kMeMB/images/4059018274-WoWScrnShot_031815_122316.jpg)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** smoldar
**Original Date:** 18.03.2015 11:28:24 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828/#comment-16553435

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** smoldar
**Original Date:** 18.03.2015 11:29:09 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828/#comment-16553449

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** eskmio
**Original Date:** 20.03.2015 20:45:02 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828/#comment-16786313

<hr>

Confirm.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 04.04.2015 05:06:30 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828/#comment-17149095

<hr>

Please let me know how this goes after update 555c572

Edit: Nvm, seems unaffected.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** smoldar
**Original Date:** 19.05.2015 08:52:24 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828/#comment-18196535

<hr>

Stil unaffected.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 19.05.2015 14:08:33 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828/#comment-18205416

<hr>

Issue #886 was marked as a duplicate of this issue.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** smoldar
**Original Date:** 21.05.2015 19:18:20 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828/#comment-18272552

<hr>

Fixed after removing this lines..
After this change is loaded all completely, objects and creatures without problems :-)

in GridNotifiers.cpp

```
#!c++

void DelayedUnitRelocation::Visit(PlayerMapType &m)
{
    for (PlayerMapType::iterator iter = m.begin(); iter != m.end(); ++iter)
    {
        Player * player = iter->getSource();
        WorldObject const *viewPoint = player->m_seer;

-        if (!viewPoint->isNeedNotify(NOTIFY_VISIBILITY_CHANGED))
-            continue;
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 21.05.2015 20:05:21 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828/#comment-18273249

<hr>

Fix #828

→ <<cset cbe949f3e8e1>>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 21.05.2015 20:05:31 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/828/#comment-18273252

<hr>

Great Work! Thank you :-)
